src/styles/theme/margin.scss

```diff
@@ -0,0 +1,16 @@
+$margin-sizes: (
+  "xsmall": $xsmall-margin,
+);
+$margin-directions: "top", "right", "bottom", "left";
+
+@each $name, $size in $margin-sizes {
+  .oc-margin-#{$name} {
+    margin: $size;
```

Comment:
This must be `margin-bottom`

Reply:
Why does it need to be `margin-bottom`?

Reply:
`uk-margin` works by only adding a margin to the bottom. Only if an element preseeds the margin, then and margin-top is added as in `* + .uk-margin…`

Otherwise, margins would start to stack on certain occasions such as these:

```html
<div class="uk-margin"></div>
<div>
   <div class="uk-margin"></div>
</div>
```